client/store/taxes/selectors.js

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { select } from '@wordpress/data';
+/**
+ * Internal dependencies
+ */
+import { ERROR } from 'store/constants';
+import { getJsonString } from 'store/utils';
+/**
+ * Returns taxes report details for a specific report query.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} query  Report query paremters
+ * @return {Object}        Report details
+ */
+function getTaxes( state, query = {} ) {
+	return get( state, [ 'taxes', getJsonString( query ) ], [] );
+}
+export default {
+	getTaxes,
+	/**
+	 * Returns true if a taxes request is pending.
+	 *
+	 * @param  {Object} state  Current state
+	 * @return {Object}        True if the `getTaxes` request is pending, false otherwise
```

Comment:
should this be Boolean instead of Object?

Reply:
Done